packages/protocol/contracts/L2/TaikoL2.sol

```diff
@@ -75,12 +87,26 @@ contract TaikoL2 is IHeaderSync {
      *
      * @param l1Height The latest L1 block height when this block was proposed.
      * @param l1Hash The latest L1 block hash when this block was proposed.
+     * @param l1Sssr The latest value of the L1 "signal service storage root".
+     * @param l1SssrProof Merkel proof for l1Sssr.
```

Comment:
There should be no need for a Merkle proof, let's do that outside the L2 smart contract and in the circuit so it's more efficient and the anchor transaction data remains small.

However not so sure where to best recover the MPT root from the block hash. I guess let's assume we can do that in the circuits already because we kind of have to do the same thing for the L2 block hash as well.